tutorials/plot_artifacts_correction_filtering.py

```diff
@@ -52,34 +52,44 @@
 # Removing power-line noise can be done with a Notch filter, directly on the
 # Raw object, specifying an array of frequency to be cut off:
 
-raw.notch_filter(np.arange(60, 241, 60), picks=picks)
+raw.notch_filter(np.arange(60, 241, 60), picks=picks, filter_length='auto',
+                 phase='zero')
 raw.plot_psd(area_mode='range', tmax=10.0, picks=picks)
 
 ###############################################################################
-# Removing power-line noise with low-pas filtering
+# Removing power-line noise with low-pass filtering
 # -------------------------------------------------
 #
 # If you're only interested in low frequencies, below the peaks of power-line
 # noise you can simply low pass filter the data.
 
-raw.filter(None, 50.)  # low pass filtering below 50 Hz
+# low pass filtering below 50 Hz
+raw.filter(None, 50., h_trans_bandwidth='auto', filter_length='auto',
+           phase='zero')
 raw.plot_psd(area_mode='range', tmax=10.0, picks=picks)
 
 ###############################################################################
 # High-pass filtering to remove slow drifts
 # -----------------------------------------
 #
-# If you're only interested in low frequencies, below the peaks of power-line
-# noise you can simply high pass filter the data.
+# To remove slow drifts, you can high pass.
+#
+# ..warning:: There can be issues using high-passes greater than 0.1 Hz
+#             (see examples in :ref:`tut_filtering_hp_problems`),
+#             so apply high-pass filters with caution.
```

Comment:
I think this renders improperly here https://1891-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_artifacts_correction_filtering.html